### DISCUSSION TITLE
#63 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0x-stan** [tog-ETH-er/togETHer#48] [General]
Agenda:

Community Updates
Fireside Chat
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/zav-cmtc-cuf

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

3个PR #961 ～ #963

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

 
- 《Reflexer Finance》by yan
  - YouTube
    - p1 https://youtu.be/5YsJADMnQWU
    - p2 https://youtu.be/okmXLIet3NI
  
    - Bilibili
      - p1 https://www.bilibili.com/video/BV1pg4y1V7Mb/
      - p2 https://www.bilibili.com/video/BV1bg4y1V7tn/
 
- 《密码学系列课程7:  GG20门限签名》by lynndell
  - Youtube:
    - p1 https://youtu.be/mfVyt047KtA
    - p2 https://youtu.be/4QVQ1oxS3rI

  - Bilibili:
    - p1 https://www.bilibili.com/video/BV11L41167iT/
    - p2 https://www.bilibili.com/video/BV1ho4y1F7os/

**Comment by luckyyang** [comment]
Rebase Community

- web3 极客日报: 最近 14 期 http://daily.rebase.network/
- 招聘: 4 个团队发布招聘 https://github.com/rebase-network/who-is-hiring/issues
- web3极客公开课（web3geekforge ） 4～5 期 https://space.bilibili.com/382886213/video

**Comment by Thegaram** [comment]
### CryptoTechNight

规划下一期CTN活动。
